### DISCUSSION TITLE
Can't figure what symbol in gutter is

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [LazyVim/LazyVim#2506] [Q&A] [answered]
What is the `m` symbol in the gutter of the image below? And if it's something that I can toggle off, how to do I do that?
<img width="442" alt="Screenshot 2024-02-05 at 10 28 09" src="https://github.com/LazyVim/LazyVim/assets/219075/d9ec5ab8-8a34-4747-9ec7-aa4abee93653">

In `Lazyvim/Neovim` is there a feature where you can hover the cursor on a symbol in the gutter to get a him os what it is?

**Comment by dpetka2001** [comment, marked as answer]
It's a mark that you have set. See `:h mark` for more info. You can delete that with `:delmarks m` or whatever is the current letter that represents the mark that you've set.
